tests/components/icloud/test_config_flow.py

```diff
@@ -182,11 +175,31 @@ def init_config_flow(hass: HomeAssistantType):
     assert result["title"] == USERNAME
     assert result["data"][CONF_USERNAME] == USERNAME
     assert result["data"][CONF_PASSWORD] == PASSWORD
-    assert result["data"][CONF_ACCOUNT_NAME] == ACCOUNT_NAME
     assert result["data"][CONF_MAX_INTERVAL] == MAX_INTERVAL
     assert result["data"][CONF_GPS_ACCURACY_THRESHOLD] == GPS_ACCURACY_THRESHOLD
 
 
+async def test_two_accounts_setup(
+    hass: HomeAssistantType, service_with_cookie: MagicMock
+):
+    """Test to setup two accounts."""
+    flow = init_config_flow(hass)
+    MockConfigEntry(
+        domain=DOMAIN, data={CONF_USERNAME: USERNAME, CONF_PASSWORD: PASSWORD}
+    ).add_to_hass(hass)
+
+    # import with username and password
+    result = await flow.async_step_import(
```

Comment:
Please use the core interfaces to set up the tests.

https://github.com/home-assistant/home-assistant/blob/d63ea198f2a9d65e2be4160f33688534744605df/homeassistant/data_entry_flow.py#L94-L96

Eg:
https://github.com/home-assistant/home-assistant/blob/d63ea198f2a9d65e2be4160f33688534744605df/tests/components/hue/test_config_flow.py#L345-L347